index.bs

```diff
@@ -2293,35 +2293,70 @@ The W3C Council</h4>
 	the [=Chair of the Council=] <em class="rfc2119">should</em> dissolve the [=Council=]
 	and call for a new one to be convened.
 
-Issue: The following text about recusals is does not relfect the latest thinking.
-However, the latest thinking isn't fully stabilized yet,
-and thus not ready to replace this.
-See <a href="https://github.com/w3c/w3process/issues/278">https://github.com/w3c/w3process/issues/278</a>
-and in particular <a href="https://github.com/w3c/w3process/issues/278#issuecomment-995868961">https://github.com/w3c/w3process/issues/278#issuecomment-995868961</a>,
-as well as <a href="https://docs.google.com/document/d/1lB8Q_pvqxfN204FQHZ8D2HObxfo36qnguif9mGpEJTU/edit#">https://github.com/w3c/w3process/issues/278#issuecomment-995868961</a>.
-
-	No one shall be both a party and an adjudicator.
-	The person making either the decision being objected to or appealed,
-	and the person making the objection or appeal,
-	<em class=rfc2119>must</em> [=recuse=] themselves.
-	Participants <em rfc=2119>should</em> [=recuse=]
-	if they believe their participation would compromise the integrity of the Council decision,
-	and <em class=rfc2119>may</em> do so at any time.
-	However, recusal is disqualification from participation,
+	Individual recusal is not used in the Council. 
+	The potential ouncil themselves decide, uniformly, which reasons for each member to serve 
+	rise to a sufficient level for that potential member to be [=dismissed=].
+	No-one is automatically dismissed.
+				
+	The team assignee <em class=rfc2119>must</em> ensure a list is developed of all potential council members, 
+	and mark against each one possible reasons that they might need to be dismissed. 
+	The W3C community,
+	including members and team, and potential council members, 
+	<em class=rfc2119>must</em> be given an opportunity to contribute possible reasons to this list.
+
+	Example reasons (none of which are diagnostic) could include:
+	<ol>
+		<li>
+			They filed the formal objection, helped author it, 
+			or work for the same organization as the author/filer
+		<li>
+			They made the decision being objected to, were editor, or a major contributor
+		<li>
+			Their business depends on the question (in either direction)
+		<li>
+			They hold sufficiently strong pre-existing opinions in either direction 
+			(e.g. a well-known proponent or objector) that their ability 
```

Comment:
```suggestion
			(e.g., a well-known proponent or objector) that their ability 
```

Reply:
not a believer in the comma-after-eg rule, myself, but OK

Reply:
not a fan either. I'll yield if others insist, but I wouldn't.

Reply:
W3C's style guide, such as it is, points to the Chicago and AP Manuals of Style, whose advice is quickly [found via Google search](https://www.google.com/search?q=comma+after+e.g.+chicago+manual+of+style):
> The Chicago Manual of Style states that i.e. and e.g. should be “confined to parentheses and notes and followed by a comma.” The AP Stylebook, whose “punctuation-pitch” leans generally to the side of “the fewer commas the better,” is pro-comma when it comes to i.e. and e.g.